recipes/nv-codev-headers/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+/* example was taken from https://www.gnu.org/ghm/2011/paris/slides/andreas-enge-mpc.pdf */
+
+#include <ffnvcodec/nvEncodeAPI.h>
+#include <iostream>
+
+int main () {
+    std::cout << "hello NVENC API version " << NVENCAPI_MAJOR_VERSION << "." << NVENCAPI_MINOR_VERSION << "\n";
```

Comment:
This only tests the header: `NVENCAPI_MAJOR_VERSION` and NVENCAPI_MAJOR_VERSION` are macros.
Please call some function such that the actual library is tested.